README.md

```diff
@@ -82,22 +82,22 @@ documents:
     - <Add more documents based on your requirement … >
 ```
 
-The clients generated from the GraphQL config file can be used in your applications to call the queries/mutations in each GraphQL document against the GraphQL API corresponding to the GraphQL schema defined in the GraphQL config file.
+The client generated from the GraphQL config file can be used in your applications to call the queries/mutations in each GraphQL document against the GraphQL API corresponding to the GraphQL schema defined in the GraphQL config file.
 
-If you want to generate multiple Ballerina clients for a given set of GraphQL documents you can use the following command.
+If you want to generate a single Ballerina clients for a given set of GraphQL documents you can use the following command.
 
 ```
 bal graphql [-i | --input] <graphql-configuration-file-path> [-o | --output] <output-location> 
 ```
 
-This will generate a separate Ballerina client for each GraphQL document with remote operations corresponding to each GraphQL query/mutation in the GraphQL document (.graphql document). 
+This will generate a Ballerina client to represent all  the GraphQL documents with remote operations corresponding to each GraphQL query/mutation in the GraphQL document (.graphql document). 
 
 For example,
 ```
 bal graphql -i graphql.config.yaml
 ```
 
-This will generate a Ballerina client stub (<document_name>_client.bal) corresponding to each GraphQL document, a util file (utils.bal) for the relevant utils methods related to the client stubs, and a schema file (types.bal) for the configured GraphQL schema. The above command can be run from anywhere on the execution path. It is not mandatory to run it from within a Ballerina project.
+This will generate a Ballerina client stub (client.bal) with all the remote operations, an util file (utils.bal) for the relevant utils methods related to the client stub, a schema file (types.bal) for the configured GraphQL schema, and a config file (config_types.bal) for all the Ballerina data types related to connector configuration. The above command can be run from anywhere on the execution path. It is not mandatory to run it from within a Ballerina project.
```

Comment:
```suggestion
This will generate a Ballerina client stub (client.bal) with all the remote operations, a util file (utils.bal) for the relevant utils methods related to the client stub, a schema file (types.bal) for the configured GraphQL schema, and a config file (config_types.bal) for all the Ballerina data types related to connector configuration. The above command can be run from anywhere on the execution path. It is not mandatory to run it from within a Ballerina project.
```